sonar-scm-svn-plugin/src/main/java/org/sonar/plugins/scm/svn/SvnScmProvider.java

```diff
@@ -103,12 +103,19 @@ public BlameCommand blameCommand() {
     return null;
   }
 
-  private static Set<Path> computeChangedPaths(Path projectBasedir, SVNClientManager clientManager) throws SVNException {
+  static Set<Path> computeChangedPaths(Path projectBasedir, SVNClientManager clientManager) throws SVNException {
     SVNWCClient wcClient = clientManager.getWCClient();
     SVNInfo svnInfo = wcClient.doInfo(projectBasedir.toFile(), null);
 
     // SVN path of the repo root, for example: /C:/Users/JANOSG~1/AppData/Local/Temp/x/y
     Path svnRootPath = toPath(svnInfo.getRepositoryRootURL());
+
+    /* the svn root path is "" (which is returned by svnkit e.g. for urls like http://svnserver/) set it to "/". to
+    avoid crashing when using Path.relativize later */
+    if(svnRootPath.equals(Paths.get(""))){
```

Comment:
This would cause the `relativize` later to fail if `svnProjectPath` is a relative path (instead of being absolute). I'm not sure if that situation is possible, but maybe it would be safer to add something like `&& svnProjectPath.isAbsolute()`?

Reply:
@dbmeneses That should be ok: the `svnProjectPath` should always be absolute.